### ISSUE TITLE
`--local-dir` is not working as expected

### ISSUE THREAD

**Issue by randex** [gabe565/gh-profile#4]
So I have one account that I use for multiple projects and another account that I use for only one private repo. I want that account to be used only in the directory of that repo.

**What I expected**:

After running `gh profile switch --local-dir` all normal `gh` commands like `gh pr list` should use the account that I've set locally instead of the global account.

**What I've got instead**:

`gh pr list` errored out with `GraphQL: Could not resolve to a Repository with the name [redacted]. (repository)`, because the repo is private and only accessible to the account that I wanted to use locally. So it seems like it used the global account. Also, when running `gh profile show` in the repo directory, it shows the global account instead of local.

**Steps I took**:

1. Installed direnv using `brew install direnv` on my Mac (macOS Monterey 12.6)
2. Ran `gh profile switch --local-dir [account name]` in the directory of that one repo.
3. Ran `gh pr list` expecting that this command will use the account I've set in step 2, but it seems like it's using the global account. Got error that it couldn't find the repo with this name (because it's private).

Am I missing something? Or did I expect the wrong thing?

**Comment by yermulnik** [comment]
Typically there should be `direnv allow` command issued in between steps 2 and 3 to let `direnv` parse and pickup `.envrc` file (if this hasn't been done yet).

**Comment by randex** [comment]
Oh, right. So here's what I did wrong, hopefully it'll help someone in the future.

1. After direnv is installed, it's important to [hook it to your shell](https://direnv.net/docs/hook.html), and I just skipped this part (don't be me). _I added `eval "$(direnv hook zsh)"` to `.zshrc` file because I use `zsh` shell._
2. Then as @yermulnik said, after you run `gh profile switch --local-dir [account name]` run another command which is `direnv allow .` or `direnv allow .envrc` so direnv can read it and load the variables from there.

After that, it started working perfectly. Thank you so much guys for this extension, it saves me so much time! 🥇
